STAT/TStatToolkit.cxx

```diff
@@ -1724,7 +1727,6 @@ TMultiGraph*  TStatToolkit::MakeStatusMultGr(TTree * tree, const char * expr, co
   //  TStatToolkit::AddStatusPad(c1, 0.30, 0.40);
```

Comment:
remove commented lines

Reply:
these lines belong to a code example, right?